src/demos/elasticsearch/App.js

```diff
@@ -0,0 +1,69 @@
+/*
+ * This file is part of React-SearchKit.
+ * Copyright (C) 2019 CERN.
+ *
+ * React-SearchKit is free software; you can redistribute it and/or modify it
+ * under the terms of the MIT License; see LICENSE file for more details.
+ */
+
+import React, { Component } from 'react';
+import { Container, Grid } from 'semantic-ui-react';
+import {
+  ReactSearchKit,
+  SearchBar,
+  EmptyResults,
+  Error,
+  ResultsLoader,
+  withState,
+} from '../../lib/components';
+import { Results } from './Results';
+import { ESSearchApi } from '../../lib/api/contrib/elasticsearch';
+
+const OnResults = withState(Results);
+
+const searchApi = new ESSearchApi({
+  url: 'http://localhost:5000/random/_search',
+  timeout: 5000,
+});
+
+export class App extends Component {
+  constructor(props) {
+    super(props);
+  }
```

Comment:
This is not needed.

Reply:
There will be more in the next PR for this, that's why I kept. If it will be not needed, I will remove, thanks for checking.